module/module.go

```diff
@@ -156,6 +157,16 @@ func NewModuleFromArgs(ctx context.Context, logger *zap.SugaredLogger) (*Module,
 	return NewModule(ctx, os.Args[1], logger)
 }
 
+// NewLoggerFromArgs can be used to create a golog.Logger at "DebugLevel" if
+// "--log-level=debug" is the third argument in os.Args and at "InfoLevel"
+// otherwise.
+func NewLoggerFromArgs(moduleName string) golog.Logger {
+	if len(os.Args) >= 3 && os.Args[2] == "--log-level=debug" {
+		return golog.NewDebugLogger(moduleName)
+	}
+	return golog.NewDevelopmentLogger(moduleName)
```

Comment:
Add a note that this is the default and the expected if someone sets log_level to "info". In other words, let's make it explicit/clear that `"log_level": "info"` is how you forcibly avoid debug from the parent.

Reply:
Sounds good; elaborated on docs for `config.Module.LogLevel` and referenced them here.